portworx-manual/temp-px-perf/px-failovertest.sh

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env bash
```

Comment:
This test fails. @NathanielRose you're checking in a lot in this PR. How about we remove these tests from the PR and you work on them in the background? What I'm seeing:

```
$ sh px-failovertest.sh 
7dba7f60-2613-4cc0-ab49-660ed7179f25
Test Topic: topic-7dba7f60-2613-4cc0-ab49-660ed7179f25
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
Created topic topic-7dba7f60-2613-4cc0-ab49-660ed7179f25.
Creating Input Message file.
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 26: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
cat: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 32: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-input-messages.txt: No such file or directory
px-failovertest.sh: line 36: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-output-messages.txt: No such file or directory
px-failovertest.sh: line 40: kill: (54493) - No such process
node "aks-agentpool-11920454-4" cordoned
Cordoning Node: aks-agentpool-11920454-4
NAME                       STATUS                     ROLES     AGE       VERSION   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
aks-agentpool-11920454-0   Ready                      agent     3h        v1.12.8   <none>        Ubuntu 16.04.6 LTS   4.15.0-1050-azure   docker://3.0.6
aks-agentpool-11920454-1   Ready                      agent     3h        v1.12.8   <none>        Ubuntu 16.04.6 LTS   4.15.0-1050-azure   docker://3.0.6
aks-agentpool-11920454-2   Ready                      agent     3h        v1.12.8   <none>        Ubuntu 16.04.6 LTS   4.15.0-1050-azure   docker://3.0.6
aks-agentpool-11920454-3   Ready                      agent     3h        v1.12.8   <none>        Ubuntu 16.04.6 LTS   4.15.0-1050-azure   docker://3.0.6
aks-agentpool-11920454-4   Ready,SchedulingDisabled   agent     3h        v1.12.8   <none>        Ubuntu 16.04.6 LTS   4.15.0-1050-azure   docker://3.0.6

Deleting kafka broker pod: px-cluster-kafka-0
pod "px-cluster-kafka-0" deleted

Deleted kafka broker is now being created on new shared node
NAME                                         READY     STATUS        RESTARTS   AGE       IP            NODE
kafkaclient-0                                1/1       Running       0          57m       10.244.2.9    aks-agentpool-11920454-1
kafkaclient-1                                1/1       Running       0          55m       10.244.3.11   aks-agentpool-11920454-0
kafkaclient-2                                1/1       Running       0          55m       10.244.1.13   aks-agentpool-11920454-4
kcluster-entity-operator-68c595b8-7vlgj      3/3       Running       0          1h        10.244.1.8    aks-agentpool-11920454-4
kcluster-kafka-0                             2/2       Running       0          1h        10.244.4.6    aks-agentpool-11920454-2
kcluster-kafka-1                             2/2       Running       0          1h        10.244.1.7    aks-agentpool-11920454-4
kcluster-kafka-2                             2/2       Running       0          1h        10.244.0.11   aks-agentpool-11920454-3
kcluster-zookeeper-0                         2/2       Running       0          2h        10.244.0.10   aks-agentpool-11920454-3
kcluster-zookeeper-1                         2/2       Running       0          2h        10.244.4.5    aks-agentpool-11920454-2
kcluster-zookeeper-2                         2/2       Running       0          2h        10.244.1.6    aks-agentpool-11920454-4
px-cluster-entity-operator-8dcbb4576-mbs28   3/3       Running       1          56m       10.244.2.10   aks-agentpool-11920454-1
px-cluster-kafka-0                           2/2       Terminating   0          57m       10.244.1.12   aks-agentpool-11920454-4
px-cluster-kafka-1                           2/2       Running       0          57m       10.244.0.13   aks-agentpool-11920454-3
px-cluster-kafka-2                           2/2       Running       0          57m       10.244.3.10   aks-agentpool-11920454-0
px-cluster-zookeeper-0                       2/2       Running       0          58m       10.244.1.11   aks-agentpool-11920454-4
px-cluster-zookeeper-1                       2/2       Running       0          58m       10.244.0.12   aks-agentpool-11920454-3
px-cluster-zookeeper-2                       2/2       Running       0          58m       10.244.3.9    aks-agentpool-11920454-0
strimzi-cluster-operator-68575878f6-x79nh    1/1       Running       0          2h        10.244.1.5    aks-agentpool-11920454-4
Kafka broker messages after cordon are written at:  ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-cordon-output-messages.txt
px-failovertest.sh: line 62: ./temp/topic-7dba7f60-2613-4cc0-ab49-660ed7179f25-cordon-output-messages.txt: No such file or directory
px-failovertest.sh: line 65: kill: (54505) - No such process

Uncordoning the node
node "aks-agentpool-11920454-4" uncordoned
```

Reply:
`px-failovertest.sh` should work with the latest commit that targets the appropriate broker service points. Can you verify?